gulpfile.js

```diff
@@ -14,6 +14,7 @@ var runSequence = require('run-sequence');
 var zip = require('gulp-zip');
 var mocha = require('gulp-mocha');
 var path = require('path');
+var walk = require('walk');
```

Comment:
Out of curiosity, can `walk` be used in "non test" scenarios to "discover" and require modules?